Events package uses ora but does not declare this dependency in package.json

- [x] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

`@truffle/events` uses `ora` but does not declare this dependency in `package.json`

## Steps to Reproduce

Create a project that requires `@truffle/events`. (In my case, I was depending on `@truffle/config`, which in turn depends on`@truffle/events` ). Try to run it.

## Expected Behavior

Things should run normally.

## Actual Results

Running the project fails with the error:

```
Error: Cannot find module 'ora'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/var/app/current/node_modules/@truffle/events/defaultSubscribers/obtain.js:1:13)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/var/app/current/node_modules/@truffle/events/defaultSubscribers/index.js:3:11)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
```

## Environment

* Operating System: various
* Ethereum client: various
* Truffle version (`truffle version`): Truffle v5.1.18 (core: 5.1.18)
* node version (`node --version`): Node v10.16.3
* npm version (`npm --version`): 6.9.0

--- a/packages/compile-solidity/package.json
+++ b/packages/compile-solidity/package.json
@@ -11,7 +11,7 @@
     "colors": "^1.1.2",
     "debug": "^4.1.0",
     "fs-extra": "^8.1.0",
-    "ora": "^3.0.0",
+    "ora": "^4.0.3",
     "original-require": "^1.0.1",
     "request": "^2.85.0",
     "request-promise": "^4.2.2",

--- a/packages/compile-solidity/package.json
+++ b/packages/compile-solidity/package.json
@@ -11,7 +11,7 @@
     "colors": "^1.1.2",
     "debug": "^4.1.0",
     "fs-extra": "^8.1.0",
-    "ora": "^4.0.3",
+    "ora": "^3.4.0",
     "original-require": "^1.0.1",
     "request": "^2.85.0",
     "request-promise": "^4.2.2",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -58,7 +58,7 @@
     "mocha": "5.2.0",
     "node-emoji": "^1.8.1",
     "node-ipc": "^9.1.1",
-    "ora": "^4.0.3",
+    "ora": "^3.4.0",
     "original-require": "1.0.1",
     "safe-eval": "^0.4.1",
     "sane": "^4.0.2",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -58,6 +58,7 @@
     "mocha": "5.2.0",
     "node-emoji": "^1.8.1",
     "node-ipc": "^9.1.1",
+    "ora": "^4.0.3",
     "original-require": "1.0.1",
     "safe-eval": "^0.4.1",
     "sane": "^4.0.2",
@@ -72,7 +73,6 @@
   },
   "devDependencies": {
     "@truffle/blockchain-utils": "^0.0.18",
-    "glob": "^7.1.2",
     "memorystream": "^0.3.1"
   },
   "publishConfig": {

--- a/packages/events/package.json
+++ b/packages/events/package.json
@@ -10,6 +10,7 @@
     "access": "public"
   },
   "dependencies": {
-    "emittery": "^0.4.1"
+    "emittery": "^0.4.1",
+    "ora": "^4.0.3"
   }
 }

--- a/packages/events/package.json
+++ b/packages/events/package.json
@@ -11,6 +11,6 @@
   },
   "dependencies": {
     "emittery": "^0.4.1",
-    "ora": "^4.0.3"
+    "ora": "^3.4.0"
   }
 }

--- a/packages/reporters/package.json
+++ b/packages/reporters/package.json
@@ -9,7 +9,7 @@
   "scripts": {},
   "dependencies": {
     "node-emoji": "^1.8.1",
-    "ora": "^3.0.0",
+    "ora": "^4.0.3",
     "web3-utils": "1.2.1"
   },
   "publishConfig": {

--- a/packages/reporters/package.json
+++ b/packages/reporters/package.json
@@ -9,7 +9,7 @@
   "scripts": {},
   "dependencies": {
     "node-emoji": "^1.8.1",
-    "ora": "^4.0.3",
+    "ora": "^3.4.0",
     "web3-utils": "1.2.1"
   },
   "publishConfig": {

